clang/lib/CodeGen/CGDebugInfo.cpp

```diff
@@ -3424,6 +3447,24 @@ llvm::DIMacroFile *CGDebugInfo::CreateTempMacroFile(llvm::DIMacroFile *Parent,
   return DBuilder.createTempMacroFile(Parent, Line, FName);
 }
 
+llvm::DILocation *
+CGDebugInfo::CreateTrapFailureMessageFor(llvm::DebugLoc TrapLocation,
+                                         StringRef FailureMsg) {
+  // Create a debug location from `TrapLocation` that adds an artificial inline
+  // frame.
+  const char *Prefix = CLANG_VERBOSE_TRAP_PREFIX;
```

Comment:
The `Prefix` should not be hardcoded. It should be a parameter to `CreateTrapFailureMessageFor` so it can be re-used by other things like `-fbounds-safety`